bin/linters.sh

```diff
@@ -2,7 +2,7 @@
 set -ex
 
 gometalinter --concurrency=4 --enable-gc --deadline=300s --disable-all\
-  --enable=aligncheck\
+  --enable=maligned\
```

Comment:
this part will conflict with seb's pr I think

Reply:
seb's PR is based on an old version of gometalinter instead of the HEAD. aligncheck has changed to maligned recently.
I think we should keep this change unless we cherrypick seb's PR as well.

Reply:
I cherry picked seb already in my PR #301 ... but maybe I can un cherry pick or resolve conflicts

Reply:
I see. I think we may still need this to pass the tests for this PR.

Reply:
it's ok, I dropped seb's change from my PR; this one is the correct minimal one and it won't conflict anymore